spring-cloud-dataflow-server-local/pom.xml

```diff
@@ -80,6 +80,11 @@
 			<version>${spring-cloud-deployer.version}</version>
 		</dependency>
 		<dependency>
+			<groupId>org.springframework.cloud</groupId>
```

Comment:
Any dependencies we need to add in this pom to me indicates that we have an upstream issue, either in `spring-cloud-starter-dataflow-server-local` or in `spring-cloud-deployer-local`.